### PR TITLE
wownero 0.11.3.0

### PR DESCRIPTION
**Pull request by iMichka**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
> It does not build with the new boost version, but at least the workarounds are now gone

any upstream issue for tracking the latest boost support?

**Comment by iMichka** [comment]
The error was:
```
[ 50%] Built target obj_checkpoints
In file included from /tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/tx_pool.cpp:40:
/tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/blockchain.h:32:10: fatal error: 'boost/asio/io_service.hpp' file not found
   32 | #include <boost/asio/io_service.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from /tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/tx_sanity_check.cpp:33:
/tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/blockchain.h:32:10: fatal error: 'boost/asio/io_service.hpp' file not found
   32 | #include <boost/asio/io_service.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
make[2]: *** [src/cryptonote_core/CMakeFiles/obj_cryptonote_core.dir/tx_verification_utils.cpp.o] Error 1
1 error generated.
make[2]: *** [src/cryptonote_core/CMakeFiles/obj_cryptonote_core.dir/tx_pool.cpp.o] Error 1
1 error generated.
make[2]: *** [src/cryptonote_core/CMakeFiles/obj_cryptonote_core.dir/tx_sanity_check.cpp.o] Error 1
In file included from /tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/cryptonote_tx_utils.cpp:40:
/tmp/wownero-20241217-43103-4xycsi/src/cryptonote_core/blockchain.h:32:10: fatal error: 'boost/asio/io_service.hpp' file not found
   32 | #include <boost/asio/io_service.hpp>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~
[ 51%] Linking CXX static library librandomx.a
```

I did not have time to check with upstream yet.

**Comment by iMichka** [comment]
I have changed this to a draft. I am wondering why the repo URL changed, but https://wownero.org still links to https://git.wownero.com ...

File: Formula/w/wownero.rb [hunk 1]
```diff
@@ -35,7 +34,7 @@ class Wownero < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "d08b8e26383ff1a7b709f65214b0f59505ca308f3b675bff55838c91de33a375"
   end
 
-  disable! date: "2025-05-11", because: "needs to use unmaintained `boost@1.85` and `protobuf@21`"
+  disable! date: "2025-05-11", because: "needs to use unmaintained `boost@1.85`"
```
Comment (Bo98) [inline Formula/w/wownero.rb:37]: IMO this shouldn't be deprecated if `monero` isn't

File: Formula/w/wownero.rb [hunk 2]
```diff
@@ -45,27 +44,14 @@ class Wownero < Formula
   depends_on "libsodium"
   depends_on "libusb"
   depends_on "openssl@3"
-  depends_on "protobuf@21"
+  depends_on "protobuf"
```
Comment (Bo98) [inline Formula/w/wownero.rb:47]: `brew linkage` isn't picking it up so looks like this doesn't support newer Protobuf yet. I wonder if the same issue applies to Monero and we didn't notice.

Comment (iMichka) [reply Formula/w/wownero.rb]: I checked and `monero` is fine.
I reverted to protobuf@21

File: Formula/w/wownero.rb (hunk 1 above)
Comment (iMichka) [reply Formula/w/wownero.rb]: Given monero and wownero is not: let's keep this deprecated for the time being.

**Review by daeho-ro — APPROVED** [review]

**Pull request merged**